ducktape/errors.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2015 Confluent Inc.
```

Comment:
Couple questions:
- Since you've added an errors file, would LoaderError (loader.py) belong here?
- I've read some sources which recommend if we roll our own errors, they should subclass a common base class, e.g. something like `DucktapeError` What do you think?